Processing for iOS/ProjectSelectionTableViewController.swift

```diff
@@ -217,15 +214,21 @@ class ProjectSelectionTableViewController: UITableViewController, UIViewControll
                     return
                 }
                 
-                let alertView = UIAlertView(title: "Error", message: message, delegate: self, cancelButtonTitle: "Cancel", otherButtonTitles: "Create")
-                alertView.alertViewStyle = .plainTextInput
-                if suggestedName != "" {
-                    alertView.textField(at: 0)?.text = suggestedName
-                }
-                alertView.show()
-                
+                self.showCreateAlert(title: message, name: suggestedName)
             }
-        }
+        }))
+        
+        alertController.addAction(UIAlertAction(title: "Cancel", style: .default
```

Comment:
The actions should be added the other way around. The "create" button is usually on the right side. Also, it should get the `.cancel` style.